### PR TITLE
Fix next_deadline calculation in handle_send_force_get_task_result

### PR DESCRIPTION
**Pull request by dybi**
resloves: https://github.com/golemfactory/concent/issues/452

### CODE REVIEW

File: concent_api/core/message_handlers.py [hunk 1]
```diff
@@ -369,13 +369,11 @@ def handle_send_force_get_task_result(client_message: message.concents.ForceGetT
         task_to_compute,
         requestor_public_key,
     )
+    maximum_download_time = calculate_maximum_download_time(client_message.report_computed_task.size, settings.MINIMUM_UPLOAD_RATE, )
```
Comment (rwrzesien) [inline concent_api/core/message_handlers.py]: Break 
parameters
into 
separate
lines.

Comment (dybi) [reply concent_api/core/message_handlers.py]: done

File: concent_api/core/tests/test_auth_get_task_result.py [hunk 2]
```diff
@@ -448,7 +448,7 @@ def test_concent_requests_task_result_from_provider_and_requestor_receives_failu
         )
 
         self.assertIsInstance(message_from_concent, message.concents.ForceGetTaskResultFailed)
-        self.assertEqual(message_from_concent.timestamp,        self._parse_iso_date_to_timestamp("2017-12-01 11:00:22"))
+        self.assertEqual(message_from_concent.timestamp,        self._parse_iso_date_to_timestamp("2017-12-01 11:00:54"))
```
Comment (kbeker) [inline concent_api/core/tests/test_auth_get_task_result.py]: If you are correcting line where was before much more spaces than, please delete this space between arguments

File: concent_api/core/tests/test_auth_get_task_result.py [hunk 3]
```diff
@@ -527,7 +527,7 @@ def test_concent_requests_task_result_from_provider_and_requestor_receives_failu
             provider_key             = self._get_encoded_provider_public_key(),
             requestor_key            = self._get_encoded_requestor_public_key(),
             expected_nested_messages = {'task_to_compute', 'report_computed_task'},
-            next_deadline            = self._parse_iso_date_to_timestamp("2017-12-01 11:00:20"),
+            next_deadline            = self._parse_iso_date_to_timestamp("2017-12-01 11:00:52"),
```
Comment (kbeker) [inline concent_api/core/tests/test_auth_get_task_result.py]: Same here

File: concent_api/core/tests/test_auth_get_task_result.py [hunk 4]
```diff
@@ -693,7 +693,7 @@ def test_concent_requests_task_result_from_provider_and_requestor_receives_failu
         )
 
         self.assertIsInstance(message_from_concent, message.concents.ForceGetTaskResultFailed)
-        self.assertEqual(message_from_concent.timestamp,                                    self._parse_iso_date_to_timestamp("2017-12-01 11:00:24"))
+        self.assertEqual(message_from_concent.timestamp,                                    self._parse_iso_date_to_timestamp("2017-12-01 11:00:54"))
```
Comment (kbeker) [inline concent_api/core/tests/test_auth_get_task_result.py]: Same here

File: concent_api/core/tests/test_auth_get_task_result.py [hunk 5]
```diff
@@ -766,7 +766,7 @@ def test_concent_requests_task_result_from_provider_and_requestor_receives_failu
             provider_key             = self._get_encoded_provider_public_key(),
             requestor_key            = self._get_encoded_requestor_public_key(),
             expected_nested_messages = {'task_to_compute', 'report_computed_task'},
-            next_deadline            = self._parse_iso_date_to_timestamp("2017-12-01 11:00:20"),
+            next_deadline            = self._parse_iso_date_to_timestamp("2017-12-01 11:00:52"),
```
Comment (kbeker) [inline concent_api/core/tests/test_auth_get_task_result.py]: And here

File: concent_api/core/tests/test_auth_get_task_result.py [hunk 6]
```diff
@@ -921,7 +921,7 @@ def test_concent_requests_task_result_from_provider_and_requestor_receives_failu
         )
 
         self.assertIsInstance(message_from_concent, message.concents.ForceGetTaskResultFailed)
-        self.assertEqual(message_from_concent.timestamp,       self._parse_iso_date_to_timestamp("2017-12-01 11:00:21"))
+        self.assertEqual(message_from_concent.timestamp,       self._parse_iso_date_to_timestamp("2017-12-01 11:00:54"))
```
Comment (kbeker) [inline concent_api/core/tests/test_auth_get_task_result.py]: And here

File: concent_api/core/message_handlers.py (hunk 1 above)
Comment (cameel) [reply concent_api/core/message_handlers.py]: I think it was fine (except for the stray comma at the end). It was not that long, it did not have to get broken into multiple lines.

File: concent_api/core/tests/test_auth_get_task_result.py (hunk 6 above)
Comment (dybi) [reply concent_api/core/tests/test_auth_get_task_result.py]: done

**Pull request merged**